lib/nve/nve-main-page.dart

```diff
@@ -121,6 +164,9 @@ class Position with ChangeNotifier {
       }
   
       _locationData = await location.getLocation();
+      _distance = await geo.distanceBetween(_locationData.latitude, _locationData.longitude, 
+                                            Provider.of<Position>(context, listen: false).latitude, 
```

Comment:
This method throws exceptions when the start location is not set yet. Please fix it